### ISSUE TITLE
Elevated D-dimers

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6100]
**Preferred term label:**
Elevated D-dimers

**Synonyms**

**Definition (free text, please give PubMed ID)**
An increased concentration of D-dimers, a marker of fibrin degradation, in the blood circulation.
PMID:19008457
Comment: The D-dimer antigen is a unique marker of fibrin degradation that is formed by the sequential action of 3 enzymes: thrombin, factor XIIIa, and plasmin. First, thrombin cleaves fibrinogen producing fibrin monomers, which polymerize and serve as a template for factor XIIIa and plasmin formation. Second, thrombin activates plasma factor XIII bound to fibrin polymers to produce the active transglutaminase, factor XIIIa. Factor XIIIa catalyzes the formation of covalent bonds between D-domains in the polymerized fibrin. Finally, plasmin degrades the crosslinked fibrin to release fibrin degradation products and expose the D-dimer antigen. D-dimer measurement has been most comprehensively validated in (1) the exclusion of venous thromboembolism (VTE) in certain patient populations and (2) the diagnosis and monitoring of coagulation activation in disseminated intravascular coagulation. Anormal D-dimer in the appropriate clinical context denotes that there is no major ongoing activation of intravascular coagulation, and serves as a reliable tool for the exclusion of VTE. 

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0040224

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:618886 
PMID: 31836668  2/2

**Your nano-attribution (ORCID)**

**Comment by LCCarmody** [comment]
FWIW, I asked D-dimer get into PR for log defs. PR:000050369

https://github.com/PROconsortium/PRoteinOntology/issues/181

**Comment by pnrobinson** [comment]
done
